certbot/certbot/_internal/cli/__init__.py

```diff
@@ -162,6 +162,17 @@ def prepare_and_parse_args(plugins: plugins_disco.PluginsRegistry, args: List[st
              " roll back those changes.  It also calls --pre-hook and --post-hook commands"
              " if they are defined because they may be necessary to accurately simulate"
              " renewal. --deploy-hook commands are not called.")
+    helpful.add(
+        ["testing", "renew", "certonly", "reconfigure"],
+        "--run-deploy-hook", action="store_true", dest="run_deploy_hook",
+        default=flag_default("run_deploy_hook"),
```

Comment:
Should "hook" here be pluralized?

```suggestion
        "--run-deploy-hooks", action="store_true", dest="run_deploy_hooks",
        default=flag_default("run_deploy_hooks"),
```

Reply:
I went back and forth on this one -- my thinking here was that the most common usage would be when it's set to test for a newly set-on-the-cli deploy hook, in which case we're just talking about the one. But pluralizing would make it more clear that it will in fact run any and all present ones, so that's probably for the best, now that I've gone and written the explanatory text.